pkg/resourcecreator/horizontalpodautoscaler/horizontalpodautoscaler_test.go

```diff
@@ -134,7 +134,7 @@ func TestHorizontalPodAutoscaler(t *testing.T) {
 
 	t.Run("should add both cpu and kafka metric when both are set", func(t *testing.T) {
 		app := fixtures.MinimalApplication()
-		topic := fmt.Sprintf("%s.mytopic", fixtures.ApplicationTeam)
+		topic := fmt.Sprintf("%s.mytopic", fixtures.ApplicationNamespace)
```

Comment:
👏

Reply:
Yes, well, now the two are the same, so it doesn't really matter.
But it does look dangerous :p